gdal/doc/source/drivers/vector/lvbag.rst

```diff
@@ -0,0 +1,58 @@
+.. _vector.lvbag:
+
+================================================================================
+Dutch Kadaster LV BAG 2.0 Extract
+================================================================================
+
+.. shortname:: LVBAG
+
+.. build_dependencies:: libexpat
+
+This driver can read XML files in the LV BAG 2.0 extract format as provided by
+the Dutch Kadaster BAG products. All LV BAG 2.0 extract products are supported.
+The driver supports all BAG layers including thorse introduced in BAG 2.0.
+
+The driver is only available if GDAL/OGR is compiled against the Expat
+library.
+
+Each extract XML file is presented as a single OGR layer. The layers are
+georeferenced in their native (EPSG:28992) SRS.
+
+Note 1 : the earlier BAG 1.0 extract is not supported by this driver.
+
+Note 2 : the driver will only read ST (Standaard Levering) extract files. Mutation
+ML (Mutatie Levering) files are not supported.
+
+Driver capabilities
+-------------------
+
+.. supports_create::
+
+.. supports_virtualio::
+
+Encoding issues
+---------------
+
+Expat library supports reading the following built-in encodings :
```

Comment:
does that need to be mentioned ? I know it is in similar pages, but in the case of this driver, if files are always in one of the below encoding, that can be removed